### PR TITLE
DOCS-3782 update to Change REST Version

### PR DESCRIPTION
**Pull request by george-hoffer**

### CODE REVIEW

**Review by wwarner-salesforce — APPROVED** [review]
Looks good!

File: modules/ROOT/pages/to-change-raml-version.adoc [hunk 1]
```diff
@@ -1,22 +1,44 @@
...
-Changing a RAML version cannot be done in Anypoint Exchange. Instead, you can change the RAML version
-either in the RAML specification itself or when you publish to Anypoint Exchange using Design Center.
+For both RAML and OAS API specifications within Exchange, you can change the version number from within Exchange using the *Add new version* button in the Exchange user interface.
 
-From the RAML specification:
+image::ex-add-new-version.png[]
+
+This opens the Publish a new asset window. 
+
+image::ex-publish-a-new-asset.png[]
```
Comment (nicolasgnr) [inline modules/ROOT/pages/to-change-raml-version.adoc:13]: The image is including SNAPSHOT's feature that will not be in production with AMF Release

**Review by nicolasgnr — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/to-change-raml-version.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/to-change-raml-version.adoc:13]: Fixed. I removed the Snapshot checkbox from the image and added it to the branch.

**Comment by nicolasgnr** [comment]
Did you guys consider adding what @damiancalabresi has written [here](https://docs.google.com/document/d/1Iw7_3cIBOWagBiXbqvr5IgK4cuA1PqVmvO4LIrTtLvE/edit#)? It's important to explain the user that relationship between the API Version and the Major Version of the Asset Version

**Review by nicolasgnr — APPROVED** [review]

**Pull request merged**